src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -61,8 +61,8 @@ class ProtectLevelCalculator(
     deliusAssessments?.rsr
       .let { rsr ->
         when {
-          rsr != null && rsr >= RsrThresholds.TIER_B_RSR.num -> 20
-          rsr != null && rsr >= RsrThresholds.TIER_C_RSR.num -> 10
+          rsr != null && rsr != RsrThresholds.NO_RSR_MAGIC_NUMBER.num && rsr >= RsrThresholds.TIER_B_RSR.num -> 20
```

Comment:
I thought we had done this too!
Is there a nice kotlin way we can reduce the duplication here?

Reply:
why yes, there is.